packages/strapi-database/lib/queries/paginated-queries.js

```diff
@@ -7,7 +7,7 @@ const createPaginatedQuery = ({ fetch, count }) => async (queryParams, ...args)
   const pagination = await getPaginationInfos(queryParams, count, ...args);
 
   Object.assign(params, paginationToQueryParams(pagination));
-  const results = await fetch(params, undefined, ...args);
+  const results = await fetch(params, ...args);
```

Comment:
That `undefined` was there for some reason?
Because that's what forces you to ignore the `populate`.

Reply:
Also extracted in it's own PR: https://github.com/strapi/strapi/pull/10561